includes/core/server/request/class-woopay-create-and-confirm-intention.php

```diff
@@ -34,4 +34,13 @@ public function set_has_woopay_subscription( $has = true ) {
 		$this->set_param( 'woopay_has_subscription', $has );
 	}
 
+	/**
+	 * Save payment method to platform.
+	 *
+	 * @param bool $save save payment method to platform.
+	 */
+	public function set_save_payment_method_to_platform( $save = true ) {
+		$this->set_param( 'save_payment_method_to_platform', $save );
```

Comment:
I guess this doesn't affect functionality, but we should probably add `save_payment_method_to_platform` to the `DEFAULT_PARAMS` const, similar to the [WooPay_Create_And_Confirm_Setup_Intention](https://github.com/Automattic/woocommerce-payments/blob/fe3e25719fac9f2817612c96ab103a1d5e8613b3/includes/core/server/request/class-woopay-create-and-confirm-setup-intention.php#L17) class.

Also noticed that in `WooPay_Create_And_Confirm_Setup_Intention` the const is named `DEFAULTS` instead of `DEFAULT_PARAMS` as in other classes, so maybe a minor suggestion for improvement.